qemu/tests/fio_windows.py

```diff
@@ -5,6 +5,7 @@
 from virttest import error_context
 from virttest import utils_misc
 
+from autotest.client.shared import utils
```

Comment:
Do not suggest use autotest modules

Reply:
@lijinlijin Has updated the patch, use the virtest.utils_misc.InterrupttedThread() replaced utils.InterrupttedThread(), thanks for you advise, pls help to review again, thanks~